### ISSUE TITLE
Added POSIX manpages build target, resulting files show `<code>` tags in verbatim. Any ideas how to address this in a style?

### ISSUE THREAD

**Issue by datenwolf** [KhronosGroup/Vulkan-Docs#850]
*TL;DR: I'd like to submit a pull request for a new build target `posixmanpages`, but before doing so want to know, if there's a simple way to get rid of markup tags getting emitted in verbatim.*

Call me old fashioned, but I largely prefer POSIX manpages over HTML or PDF. So I added a new target to the Makefile, `posixmanpages`, which looks about like this:

```
# These targets are Unix man pages

# The recursive $(MAKE) is an apparently unavoidable hack, since the
# actual list of man page sources isn't known until after
# man/apispec.txt is generated.
posixmanpages: man/apispec.txt
	$(MAKE) buildposixmanpages

MANPOSIXDIR  = $(OUTDIR)/man/man3
MANPOSIX     = $(MANSOURCES:$(MANDIR)/%.txt=$(MANPOSIXDIR)/%.3)
buildposixmanpages: $(MANPOSIX)

$(MANPOSIXDIR)/%.3: KATEXDIR = ../../katex
$(MANPOSIXDIR)/%.3: $(MANDIR)/%.txt $(MANCOPYRIGHT) $(GENINCLUDE) $(GENDEPENDS) katexinst
	$(QUIET)$(MKDIR) $(MANPOSIXDIR)
	$(QUIET)$(ASCIIDOC) -b manpage $(ADOCOPTS) $(ADOCPOSIXOPTS) -d manpage -o $@ $<

```

However in the generated manpages `<code>` tags show up in verbatim. E.g. for `vkCmdDraw`

> ```
> VKCMDDRAW(3)                                                      VKCMDDRAW(3)
> 
> NAME
>        vkCmdDraw - Draw primitives
> 
> C SPECIFICATION
>        To record a non-indexed draw, call:
> 
>            void vkCmdDraw(
>                VkCommandBuffer                             commandBuffer,
>                uint32_t                                    vertexCount,
>                uint32_t                                    instanceCount,
>                uint32_t                                    firstVertex,
>                uint32_t                                    firstInstance);
> 
> PARAMETERS
>        •   <code>commandBuffer</code> is the command buffer into which the
>            command is recorded.
> 
>        •   <code>vertexCount</code> is the number of vertices to draw.
> 
>        •   <code>instanceCount</code> is the number of instances to draw.
> 
>        •   <code>firstVertex</code> is the index of the first vertex to draw.
> 
>        •   <code>firstInstance</code> is the instance ID of the first instance
>            to draw.
> 
> DESCRIPTION
>        When the command is executed, primitives are assembled using the
>        current primitive topology and <code>vertexCount</code> consecutive
>        vertex indices with the first <code>vertexIndex</code> value equal to
>        <code>firstVertex</code>.  The primitives are drawn
>        <code>instanceCount</code> times with <code>instanceIndex</code>
>        starting with <code>firstInstance</code> and increasing sequentially
>        for each instance.  The assembled primitives execute the bound graphics
>        pipeline.
> ```

I'm pretty clueless about *asciidoctor*, so I wonder if there's some easy fix in the styles or configuration to treat them differently.

**Comment by oddhack** [comment]
The answer might be as simple as our using some passthrough construct that passes literal tags to more than just the html5 outputs, or it could be something specific to this target, which we're not familiar with.

We chose a long time back to not support manpage targets or host them on the registry. That's mostly because there was very little demand seen from developers, and also there are some parts of the spec that are intrinsically extremely difficult to translate correctly to a text format. We discuss this from time to time, most recently in response to https://github.com/KhronosGroup/Vulkan-Docs/pull/839, and still don't have a desire to revisit this decision.

That said, we'd welcome someone such as yourself cloning the repo and developing a functioning reference page solution. So you might want to comment over on !839, and perhaps collaborate with Ryp to provide this as a downstream resource for interested parties. We'd also be happy to provide a link to such a resource from the registry index page, once it's ready for use. Sorry not to be more helpful, this just isn't in scope for what we have resources to maintain.

**Comment by datenwolf** [comment]
Alright. I totally understand the reasoning – and also wonder why I didn't see / overlooked #839 when searching the issue tracker. I'll give the fork & 3rd party host some thought. But it'd be great if people (more) knowledgeable with asciidoc and asciidoctor could give me a boost, because a) I want to learn that stuff but b) I'm swamped with other work, so would be really grateful for any hand holding I can get.

I'm closing this issue, because duplicate of #839 and I kindof got my answer.

**Comment by krOoze** [comment]
@datenwolf To answer your original question, it should be as easy as going to the `Vulkan-Docs/config/vulkan-macros/extension.rb` and modify any line that contains `<code>` with 

```
if document.backend == 'manpage'
    return yourmanpagecompatiblestring
else
   return theoriginalstringwithcodetags
end
```
